ou_dedetai/wine.py

```diff
@@ -434,11 +434,21 @@ def set_win_version(exe, windows_version):
         wait_pid(process)
 
 
-def install_icu_data_files(app=None):
+def enforce_icu_data_files(app=None):
     repo = "FaithLife-Community/icu"
     json_data = network.get_latest_release_data(repo)
     icu_url = network.get_first_asset_url(json_data)
-    # icu_tag_name = utils.get_latest_release_version_tag_name(json_data)
+    icu_latest_version = network.get_tag_name(json_data).lstrip('v')
+
+    # This file with the tag name of the downloaded release. If it doesn't match latest override the ICU files then write this file
+    icu_version_path = Path(f"{config.WINEPREFIX}/drive_c/windows/globalization/ICU/{repo.replace('/','_')}_Version.txt")
```

Comment:
Should we store this in `"~/.local/state/FaithLife-Community/` instead?

Reply:
Seems reasonable to me, I'll make the change shortly

Reply:
Fixed